drivers/backup/auth.go

```diff
@@ -495,7 +495,8 @@ func GetPxCentralAdminCtx() (context.Context, error) {
 // UpdatePxBackupAdminSecret updating "px-backup-admin-secret" token with
 // "px-central-admin" token
 func UpdatePxBackupAdminSecret() error {
-	pxCentralAdminToken, err := GetPxCentralAdminToken()
+	//pxCentralAdminToken, err := GetPxCentralAdminToken()
```

Comment:
Remove commented code

Reply:
Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint